Author: Wolfgang Schweer <wschweer@arcor.de>
Description: Adjust Inkscape options for SVG conversion
 Inkscape removed the -z option (--without-gui shortcut) and deprecated the
 --without-gui, --export-png and --export-pdf options (among others).
 Now, --export-filename and/or --export-type are needed, --without-gui is the
 default for commandline usage.   
Bug-Debian: https://bugs.debian.org/1033547
--- a/lib/dbtexmf/core/imagedata.py
+++ b/lib/dbtexmf/core/imagedata.py
@@ -181,7 +181,7 @@ class FigConverter(ImageConverter):
 class SvgConverter(ImageConverter):
     def __init__(self, imgsrc, imgdst="", docformat="", backend=""):
         ImageConverter.__init__(self, imgsrc="svg", imgdst=imgdst)
-        self.add_command(["inkscape", "-z", "-D", "--export-%(dst)s=%(output)s",
+        self.add_command(["inkscape", "-D", "--export-filename=%(output)s",
                           "%(input)s"])
 
 
